app/javascript/src/LoginPage/loginForms/FormGroups.jsx

```diff
@@ -8,14 +8,17 @@ import {
 
 type InputProps = {
   isRequired: boolean,
-    label: string,
-    fieldId: 'string',
-    isValid: boolean,
-    name: string,
-    value: string,
-    onChange: () => void,
-    autoFocus?: string,
-    ariaInvalid?: boolean
+  label: string,
+  fieldId: 'string',
+  isValid: boolean,
+  name: string,
+  value: string,
+  onChange?: () => void,
+  onBlur?: () => void,
+  autoFocus?: string,
+  ariaInvalid?: boolean,
+  isPasswordConfirmation?: boolean,
+  passwordDoesntMatch?: boolean
```

Comment:
This field also seems too restrictive. I think it would be better to pass a string and have more control.